tests/hardware_reboot/testdata/hardware_usb_reboot.txt

```diff
@@ -57,27 +66,27 @@ for i in `seq 20`
 do
  sleep 20
  # Test SSH-access to container
- echo $i\) ssh -o ConnectTimeout=10 -o StrictHostKeyChecking=no -o PasswordAuthentication=no -i {{EdenConfig "eden.root"}}/tests/eclient/image/cert/id_rsa -p $port root@$HOST grep Ubuntu /etc/issue
- ssh -o ConnectTimeout=10 -o StrictHostKeyChecking=no -o PasswordAuthentication=no -i {{EdenConfig "eden.root"}}/tests/eclient/image/cert/id_rsa -p $port root@$HOST grep Ubuntu /etc/issue && break
+ echo $i\) ssh -o ConnectTimeout=10 -o StrictHostKeyChecking=no -o PasswordAuthentication=no -i {{EdenConfig "eden.root"}}/tests/eclient/image/cert/id_rsa -p $port ubuntu@$HOST grep Ubuntu /etc/issue
```

Comment:
Please change all lines from `{{EdenConfig "eden.root"}}/tests/eclient/image/cert/id_rsa` to `{{EdenConfig "eden.tests"}}/eclient/image/cert/id_rsa`

Reply:
Ok, I will change.